### PR TITLE
Support for xsoar 8/xsiam configuration

### PR DESCRIPTION
**Pull request by barryyosi-panw**
## Status

- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues

fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-6603)

## Description

A few sentences describing the overall goals of the pull request's commits.

## Screenshots

Supporting XSOAR NG/ XSIAM env vars setting. 

## Does it break backward compatibility?

- [ ] Yes
  - Further details:
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,7 +87,7 @@ If the `license/cla` status check remains on *Pending*, even though all contribu
 
 * `npm install`
 * `npm run compile`
-* `pip demisto-sdk`
+* `pip install demisto-sdk`
```
Comment (ilaner) [inline README.md:90]: nice catch

File: src/devEnvs.ts [hunk 2]
```diff
@@ -306,10 +325,31 @@ export async function configureDemistoVars(): Promise<void> {
         env["DEMISTO_API_KEY"] = apiKey;
       }
     });
+  
+  // XSIAM Auth ID
+  if (configuredPlatform === Platform.XSOAR8_XSIAM){
+    await vscode.window
+      .showInputBox({
+        title: "XSIAM Auth ID",
+        ignoreFocusOut: true,
```
Comment (ilaner) [inline src/devEnvs.ts:334]: can you add ignorefocusout to all input boxes here?

Comment (barryyosi-panw) [reply src/devEnvs.ts:339]: done

**Review by ilaner — APPROVED** [review]
good job!

**Comment by barryyosi-panw** [comment]
@ilaner Please force merge

**Pull request merged**